mteb/tasks/Audio/Clustering/eng/VoxCelebClustering.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import annotations
+
+from mteb.abstasks.Audio.AbsTaskAudioClustering import AbsTaskAudioClustering
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+# ASSUMED VOXCELEB IN CLASSIFICATION TASK WAS ACCURATE.
+
+
+class VoxCelebClustering(AbsTaskAudioClustering):
+    label_column_name: str = "label_id"
+    metadata = TaskMetadata(
+        name="VoxCelebClustering",
+        description="Clustering task based on the VoxCeleb dataset for sentiment analysis, clustering by positive/negative sentiment.",
+        reference="https://huggingface.co/datasets/DynamicSuperb/Sentiment_Analysis_SLUE-VoxCeleb",
+        dataset={
+            "path": "DynamicSuperb/Sentiment_Analysis_SLUE-VoxCeleb",
+            "revision": "554ad4367e98b7c6f4d4d9756dc6bbdf345e042e",
+        },
+        type="AudioClustering",
+        category="a2a",
+        eval_splits=["train"],
```

Comment:
I was only able to see the test split on the [huggingface dataset page](https://huggingface.co/datasets/DynamicSuperb/Sentiment_Analysis_SLUE-VoxCeleb). 
```suggestion
        eval_splits=["test"],
```